.circleci/config.yml

```diff
@@ -2,65 +2,22 @@
 #
 # Check https://circleci.com/docs/2.0/language-python/ for more details
 #
+
+version: 2
```

Comment:
I think a lot of these changes are because I merged "main" into my branch. Can we merge main into the "major-release" branch?

And apologies if that's not standard practice

Reply:
Yeah, it's not super well defined at the moment. But it should be safe for anyone to merge main into major-release whenever they need to.

Reply:
Sure

Reply:
Ok should be g2g